app/controllers/v3/service_credential_bindings_controller.rb

```diff
@@ -26,14 +27,32 @@ def index
   end
 
   def show
-    message = ServiceCredentialBindingsShowMessage.from_params(query_params)
+    message = ServiceCredentialBindingShowMessage.from_params(query_params)
     invalid_param!(message.errors.full_messages) unless message.valid?
 
     ensure_service_credential_binding_is_accessible!
 
     render status: :ok, json: serialized(message)
   end
 
+  def create
+    message = ServiceCredentialBindingCreateMessage.new(hashed_params[:body])
+    unprocessable!(message.errors.full_messages) unless message.valid?
+
+    service_instance = VCAP::CloudController::ServiceInstance.first(guid: message.service_instance_guid)
+    unprocessable!("Service instance '#{message.service_instance_guid}' not found") unless !service_instance.blank? && can_read_from_space?(service_instance.space)
```

Comment:
Route bindings create return a slightly different message from the this. 

> `The service instance could not be found: #{guid}"`

Would be nice to have consistent messages from both endpoints, but I'm not sure which one is the most consistent